src/index.html

```diff
@@ -3,10 +3,378 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/main.scss">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header-logo">
+        <a href="https://www.google.com/">
+          <img src="images/logo.png" alt="MOYO_LOGO">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link"href="#">samsung</a>
+          </li>
+          <li class="nav__item">
+            <a  class="nav__link" href="#">Smartphones</a>
```

Comment:
Don;t forget about code style, fix in all code
```suggestion
      <div class="header-logo">
        <a href="https://www.google.com/">
          <img src="images/logo.png" alt="MOYO_LOGO">
        </a>
      </div>
      
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active" href="#">apple</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link"href="#">samsung</a>
          </li>
          
          <li class="nav__item">
            <a  class="nav__link" href="#">Smartphones</a>
```